server/risk_scores/risk_assessment.py

```diff
@@ -37,25 +39,61 @@ class RiskAssessment(Enum):
 assessment_ranges.sort(key=lambda range: range[0])
 
 
-def query_mongo(initials: str, timeframe: int = 12):
-    """
-    
-    Parameters:
-        timeframe: Months from the current date to look backward in time.
-    """
+def get_incident_similarity(prev_incident: submit_schema.Form, current_incident: submit_schema.Form):
+    similarity = 0
+    coefficient = 1
+
+    similar_fields = {
+        'client_secondary': 1,
+        'location': 1,
+        'incident_type_primary': 2,
+        'incident_type_secondary': 1,
+        'child_involved': 1,
+        'program': 1,
+    }
+
+    for field, score in similar_fields.items():
+        if getattr(current_incident, field) == getattr(prev_incident, field):
+            similarity += score
+
+    return coefficient * similarity
+
+
+def get_incident_recency(prev_incident: submit_schema.Form, current_incident: submit_schema.Form, timeframe):
+    # TODO: Standardize all dates in the databse
```

Comment:
I believe there are 2 formats the dates are stored in the database currently, 2020-09-15T10:53:00.000+00:00 (Date) vs 2020-02-26 23:50 (String) which we need to standardize.

Reply:
I'll make a ticket